test_unstructured_ingest/test-ingest-slack.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+cd "$SCRIPT_DIR"/.. || exit 1
+
+if [ -z "$SLACK_CHANNELS" ] || [ -z "$SLACK_TOKEN" ]; then
+   echo "Skipping Slack ingest test because SLACK_CHANNELS and SLACK_TOKEN env vars are not set."
+   exit 0
+fi
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+        --slack-channels "${SLACK_CHANNELS}" \
```

Comment:
actually, SLACK_CHANNELS can be hardcoded in this script since the name is visible in the expected output file anyway (not a secret, unlike the token).